openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1220,7 +1220,7 @@ This specification defines the following Credential Issuer Metadata:
       * `name`: REQUIRED. String value of a display name for the Credential.
       * `locale`: OPTIONAL. String value that identifies the language of this object represented as a language tag taken from values defined in BCP47 [@!RFC5646]. Multiple `display` objects MAY be included for separate languages. There MUST be only one object for each language identifier.
       * `logo`: OPTIONAL. A JSON object with information about the logo of the Credential with a following non-exhaustive list of parameters that MAY be included:
-          * `url`: OPTIONAL. URL where the Wallet can obtain a logo of the Credential from the Credential Issuer.
+          * `uri`: OPTIONAL. String value that contains a URI where the Wallet can obtain a logo of the Credential from the Credential Issuer. Wallet needs to determine the scheme, since the URI value could use `https:` scheme, `data:` scheme, etc.
           * `alt_text`: OPTIONAL. String value of an alternative text of a logo image.
```

Comment:
```suggestion
          * `alt_text`: OPTIONAL. String value of an alternative text of a logo image.
          * `media_type`: RECOMMENDED. Media type of the logo image. It MUST be one of the types listed in the IANA Media Types registry [@IANA.MediaTypes]. Eg: `image/svg+xml` or `image/png`.
```

Reply:
Is there therefore something to be added in the security consideration, something like

> Incorrect or mismatched media types can lead to security vulnerabilities. For example, if an application expects a file to be a certain type based on its media type, but the file's actual content is different, this could lead to issues like code injection attacks. By correctly specifying the media type, applications can better validate and sanitize incoming data, reducing the risk of such attacks.

Reply:
an example of static file type inspection is using libmagic
https://linux.die.net/man/3/libmagic

Reply:
I am more in favor of changing the structure to the array. with this approach, logo will have to be array of objects resulting in additional complexity. pointing to RFC2397 for data: scheme is sufficient IMO

Reply:
I can buy this @awoie :+1: